tests/distributed/test_sequence_parallel.py

```diff
@@ -288,12 +289,15 @@ def _compare_sp(
     "RedHatAI/Meta-Llama-3.1-8B-Instruct-FP8": SPTestSettings.fp8_quant(),
 }
 
-SP_TEST_MODELS = [
+SP_TEST_MODELS = {
     # TODO support other models
     # [LANGUAGE GENERATION]
-    "meta-llama/Llama-3.2-1B-Instruct",
-    "RedHatAI/Meta-Llama-3.1-8B-Instruct-FP8"
-]
+    "meta-llama/Llama-3.2-1B-Instruct":
+    True,
+    # FP8 reduction requires sm90 or higher
+    "RedHatAI/Meta-Llama-3.1-8B-Instruct-FP8":
+    torch.cuda.get_device_capability() >= (9, 0),
```

Comment:
@tlrmchlsmth @cascade812 @ProExpertProg upgrading to PyTorch 2.8 causes this test to fail with "FP8 reduction requires sm90 or higher". Is this expected?

Reply:
Huh, did something change in PyTorch around fp8 on sm89?

Reply:
As the [error](https://buildkite.com/vllm/ci/builds/23458#0197ecce-8e56-4552-ac08-0da86736fd50/7-3025) seems to come from [NCCL](https://github.com/NVIDIA/nccl/blob/7c12c627c62ef4e5a2485777a8d9dce58f3f562f/src/enqueue.cc#L2339), this might be due to NCCl update to `nvidia-nccl-cu12==2.27.3`.

@atalman @Skylion007 Should we cherry pick https://github.com/pytorch/pytorch/pull/157108 to 2.8 and see if it helps?

Reply:
It's working now after rebasing https://buildkite.com/vllm/ci/builds/24144#01981483-8d49-4988-843b-3d3903321dd4/7-4972, so it's probably an issue from vLLM that has been fixed.